python/src/MultiStart_doc.i.in

```diff
@@ -0,0 +1,97 @@
+%feature("docstring") OT::MultiStart
+"Multi start optimization algorithm.
+
+The idea is to generate N points and run a solver starting from these
+random points to avoid local minima.
```

Comment:
Not "the idea", but: "the algorithm proceeds as follow". It is maths here, no blabla ;-)!